src/ngraph/runtime/cpu/cpu_backend.cpp

```diff
@@ -114,35 +122,27 @@ bool runtime::cpu::CPU_Backend::call(shared_ptr<Function> func,
     return rc;
 }
 
-void runtime::cpu::CPU_Backend::remove_compiled_function(shared_ptr<Function> func)
-{
-    m_function_map.erase(func);
-}
-
-void runtime::cpu::CPU_Backend::enable_performance_data(shared_ptr<Function> func, bool enable)
+void runtime::cpu::CPU_Backend::remove_compiled_function(shared_ptr<Executable> exec)
```

Comment:
Same here. It looks like we want a reference since we are not extending the ownership.